webpack.config.js

```diff
@@ -79,13 +84,20 @@ if (targetEnvironment === "production") {
 }
 
 // telemetry configuration
-let telemetryAppId;
-if (ui === "firefox-infobar-ui") {
-  telemetryAppId = "org-mozilla-bergamot";
-} else if (ui === "cross-browser-ui") {
-  telemetryAppId = "org-mozilla-bergamot-cross-browser";
-} else {
-  throw Error("Could not determine telemetry app id");
+let telemetryAppId = "org-mozilla-bergamot-test";
+
+// disabled for cross-browser until we figure out which IDs are more suitable
+// and get data collection review approval for these channels of distribution
+if (targetEnvironment === "production") {
+  if (ui === "firefox-infobar-ui") {
+    telemetryAppId = "org-mozilla-bergamot";
+  } else if (ui === "cross-browser-ui" && targetBrowser === "firefox") {
```

Comment:
Since this divide between firefox and chrome is not relevant at this stage, use a simpler if-tree, like the one that currently exists in main. Enough to comment out https://github.com/mozilla-extensions/bergamot-browser-extension/pull/63/files#diff-1fb26bc12ac780c7ad7325730ed09fc4c2c3d757c276c3dacc44bfe20faf166fL86

Reply:
removed them for now